### PR TITLE
loaded empty project

### PR DESCRIPTION
**Pull request by trapvpack**

### CODE REVIEW

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: GitHub Actions Demo
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on:
+  push:
+    branches:
+      - 'main'
+      - '30'
```
Comment (l3r8yJ) [inline .github/workflows/github-actions.yml]: remove please
```suggestion
```

File: .github/workflows/github-actions.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        run: npm run format
+      - name: type checking
+        run: npm run type-check
+      - name: unit testing
+        run: npm run test:unit
+      - name: build
+        run: npm run build
+
+
+
```
Comment (l3r8yJ) [inline .github/workflows/github-actions.yml]: empty lines, remove

File: .github/workflows/github-actions.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+name: GitHub Actions Demo
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on:
+  push:
```
Comment (l3r8yJ) [inline .github/workflows/github-actions.yml:4]: absence of `pull_request` trigger

**Review by l3r8yJ — CHANGES_REQUESTED** [review]
@trapvpack take a look at my comments below, please

**Review by l3r8yJ — APPROVED** [review]
@trapvpack looks good to me! @SergeyShapovaIov take a look, please

**Review by ssh4rd — APPROVED** [review]
All right

**Pull request merged**